flask_resty/authorization.py

```diff
@@ -14,6 +14,9 @@ def authorize_request(self):
     def filter_query(self, query, view):
         raise NotImplementedError()
 
+    def authorize_create_item(self, item):
+        raise NotImplementedError()
```

Comment:
Should this default to `pass`? That would make this change non-breaking, but it feels sort of weird.

Reply:
good question. I vote for the breaking change, all our application will catch this in the unit test anyway

Reply:
Yeah, that's my inclination right now.